driver-core/src/main/java/com/datastax/driver/core/querybuilder/Clause.java

```diff
@@ -111,6 +111,78 @@ boolean containsBindMarker() {
         }
     }
 
+    static class ContainsClause extends AbstractClause {
+
+        private final List<Object> values;
+
+        ContainsClause(String name, List<Object> values) {
```

Comment:
Since it is required for values to be a List of size 1, why not change this to `Object value` instead?  Same applies for Contains key clause.